src/N3Store.js

```diff
@@ -211,6 +211,14 @@ export default class N3Store {
 
   // ## Public methods
 
+  // ### `add` adds the specified quad to the dataset.
```

Comment:
For the new methods, I copied the documentation from the RDF/JS DatasetCore spec and tweaked it a bit. Please let me know if there is anything you want me to change.